src/Illuminate/Broadcasting/BroadcastManager.php

```diff
@@ -81,12 +82,16 @@ public function socket($request = null)
             return $request->header('X-Socket-Id');
         }
 
+        if ($request->hasCookie('io')) {
+            return $request->cookie('io');
+        }
+
         if (! $request->hasSession()) {
             return;
         }
 
         return $this->app['cache']->get(
-            'pusher:socket:'.$request->session()->getId()
+            'realtime:socket:'.$request->session()->getId()
```

Comment:
yeah, makes sense, ive just updated that now. also of note, ive updated my rebound package to have csrf, and socket remembering functionality, it now works as-is with this setup.